components/breadcrumb/overview.md

```diff
@@ -105,6 +105,8 @@ The Breadcrumb provides the following features:
 
 * `Height` - the height of the Breadcrumb component.
 
+* `Size` - the size of the Breadcrumb component. You can set it to a member of the `Telerik.Blazor.ThemeConstants.Breadcrumb.Size` class. The default value is `Medium`.
```

Comment:
Use an [em dash](https://docs.telerik.com/style-guide/lists) for the definition lists. Make sure you update all list items.
```suggestion
* `Size`&mdash;the size of the Breadcrumb component. You can set it to a member of the `Telerik.Blazor.ThemeConstants.Breadcrumb.Size` class. The default value is `Medium`.
```